src/test/java/com/vinted/flink/bigquery/BigQueryBufferedSinkTest.java

```diff
@@ -102,7 +102,7 @@ private Function<StreamExecutionEnvironment, DataStream<Rows<String>>> pipeline(
     private Function<StreamExecutionEnvironment, DataStreamSink<Rows<String>>> withBigQuerySink(MockJsonClientProvider<String> mockClientProvider, Function<StreamExecutionEnvironment, DataStream<Rows<String>>> pipeline) {
         var sink = BigQueryStreamSink.<String>newBuilder()
                 .withClientProvider(mockClientProvider)
-                .withDeliveryGuarantee(DeliveryGuarantee.AT_LEAST_ONCE)
+                .withDeliveryGuarantee(DeliveryGuarantee.EXACTLY_ONCE)
```

Comment:
why did you changed it ?

Reply:
oh, it's beffered test, ignore

Reply:
Cuase it is buffered sync in should have exactly once set

Reply:
it was an issue with the test